src/path_cache/chunk.rs

```diff
@@ -208,6 +210,40 @@ impl Chunk {
         }
     }
 
+    pub fn add_nodes_parallel<N: Neighborhood + Sync>(
```

Comment:
```suggestion
    #[cfg(feature = "parallel")]
    pub fn add_nodes_parallel<N: Neighborhood + Sync>(
```
To avoid "unused" warnings